src/components/CubeAnalysisNavbar.js

```diff
@@ -45,27 +45,27 @@ const CubeAnalysisNavBar = ({
 
   if (draftFormats) {
     dropdownElement = (
-      <Row>
-        <Col>
-          <h5>{formatId >= 0 ? draftFormats[formatId].title : 'Default Draft Format'}</h5>
-        </Col>
-        <Col>
-          <UncontrolledDropdown nav inNavbar>
+      <Col>
```

Comment:
Why is `<Col>` wrapping `<Row>`? Should be the other way around. There is no point to having a `<Row>` with no `<Col>` children

Reply:
There does seem to be a slight visual difference when taking them out, though it looks better without the row tags I think. Removing them adds more of a margin between the text and the left edge of the enclosing box. I'll update the PR with the Row tags removed.